Mathlib/RingTheory/Flat/EquationalCriterion.lean

```diff
@@ -263,7 +263,7 @@ theorem exists_factorization_of_comp_eq_zero_of_free [Flat R M] {K N : Type u}
   have (K' : Submodule R K) (hK' : K'.FG) : ∃ (κ : Type u) (_ : Fintype κ) (a : N →ₗ[R] (κ →₀ R))
       (y : (κ →₀ R) →ₗ[R] M), x = y ∘ₗ a ∧ K' ≤ LinearMap.ker (a ∘ₗ f) := by
     revert N
-    apply Submodule.fg_induction (P := _) (N := K') (hN := hK')
+    apply Submodule.fg_induction (N := K') (hN := hK')
```

Comment:
This is still `apply`, but `Submodule.fg_induction` isn't marked `elab_as_elim` and the `(P := _)` is unnecessary regardless.

Making it be `elab_as_elim` seems like it might need to wait until the next Lean release.